src/types/payment_methods_type.py

```diff
@@ -1,9 +1,17 @@
-from graphene import Boolean, ObjectType
+from graphene import Boolean, Enum, ObjectType
+
+class PaymentMethodsEnum(Enum):
+  BRB = 0
+  CASH = 1
+  CORNELL_CARD = 2
+  CREDIT = 3
+  MOBILE = 4
+  SWIPES = 5
```

Comment:
do Enums need corresponding integer values?

Reply:
The Graphene documentation for Enums says it needs some constant value and it's just a wrapper for the standard Python enum which I think always needs some value (which can be a string with the corresponding name)